meta/saimetadatautils.c

```diff
@@ -110,6 +110,25 @@ const sai_attr_metadata_t* sai_metadata_get_attr_metadata(
     return NULL;
 }
 
+static int sai_metadata_attr_id_name_cmp(_In_ const char * str1, _In_ const char * str2, _In_ size_t num)
+{
+    char c1 = '\0';
+    char c2 = '\0';
+    while (num > 0)
+    {
+        c1 = *str1++;
+        c2 = *str2++;
+        if (sai_serialize_is_char_allowed(c1) || c1 != c2) {
```

Comment:
add empty line before IF, put { in next line, add empty line after each }

Reply:
ACK